### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/dirdbm.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4879]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4879|
|Type|enhancement|
|Created|2011-02-11 05:46:36Z|
|Branch|https://github.com/twisted/twisted/tree/dirdbm-sphinx-4879|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the dirdbm howto

<details><summary>Searchable metadata</summary>

```
trac-id__4879 4879
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_dirdbm_sphinx_4879 branches/dirdbm-sphinx-4879
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297403196000000 1297403196000000
changetime__1300074998000000 1300074998000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30691]) Branching to 'dirdbm-sphinx-4879'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30692]) Merge the dirdbm parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4879](https://github.com/twisted/twisted/issues/4879)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_7"></a>|@khorn set owner to @lvh|
|-|-|

Looks good.  Merge it.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_8"></a>|@lvh set status to `closed`|
|-|-|

(In [31000]) Merge dirdbm.xhtml changes from dirdbm-sphinx-4879.


Author: screwtape, jdb
Reviewer: khorn
Fixes: [#4879](https://github.com/twisted/twisted/issues/4879)
  
"Makes Sphinx looks gooder." -- JP
